src/Screens/courses/courseCollection/CourseCollection.jsx

```diff
@@ -39,30 +45,27 @@ const CourseUserCard = ({ name, data, btnName, subName }) => {
   return (
     <>
       <h4 className='courses-users-collection-header'>{name}</h4>
-      <div className={data.length === 1
-        ? 'courses-collection-main-container single'
-        : data.length === 2
-          ? 'courses-collection-main-container double'
-          : 'courses-collection-main-container'}
-      >
+      <CardLayout data={data}>
```

Comment:
item size is covering whole screen irrespective of the no. of card to be shown for that screen size. screensize should decide no of item per row and if less item is should show empty space
![image](https://user-images.githubusercontent.com/2909490/121551392-54358780-ca2f-11eb-84d1-937d78c0fe60.png)